### DISCUSSION TITLE
DOI links broken?

### DISCUSSION THREAD

**Discussion by jarmitage** [knowledgefutures/pubpub#1272] [Support] [answered]
Example post: https://hybrid-livecode.pubpub.org/pub/sicchio/release/1
Taken from this event: https://hybrid-livecode.pubpub.org/workshop2020

DOI from above post: https://doi.org/10.21428/40e92d8f.9ee35bfc
Redirects to (not found): https://hybrid-livecode.pubpub.org/nime2020

Are we doing something wrong?

**Comment by gabestein** [comment, marked as answer]
Hello! Have you tried re-registering the DOI from the workshop2020 collection? My guess is the URL for the collection changed from `/nime2020` to `/workshop2020` and just hasn't been re-deposited yet. To fix that, all you should need to do is visit the collection and click redeposit in the metadata section.

It can take a bit for that to go through (and Crossref's queue seems to be particular busy today), but if it's not working by the end of the day, let me know!

**Reply by jarmitage** [reply]
Sounds about right. Let me check. Thanks!

**Reply by hlj24** [reply]
@gabestein I am also having trouble with DOI links. I have created a new release of several different Pubs and resubmitted the deposits, but after trying several times and waiting a few days between each try, the links are still broken. I haven't changed the URL slug on these, and I haven't selected a "Content Version" on these. Do you have any insight on this? Here's an example:

https://handbook.pubpub.org/pub/case-digital-himalaya

https://doi.org/10.21428/51bee781.028ec770

Thanks!

**Reply by gabestein** [reply]
Looking at the logs, it's complaining that the journal title has changed. Did you recently change the name of the Community, by chance? Unfortunately, Crossref doesn't like this, and we'll have to file a support ticket for them to change it manually.

**Reply by hlj24** [reply]
Ah, yes, that would be why. We realized that the title field didn't include the project's full name. That probably means there are a number of broken DOIs. Would it help if I sent a list of those?

**Reply by gabestein** [reply]
Unfortunately, I think those'll need to be resubmitted — but I've written in to Crossref to request them to change the title, which will allow you to make them. I'll let you know when that's been changed!

**Reply by gabestein** [reply]
Ok, the title has been updated! You should be free to re-deposit now. Let me know if you need anything else.

**Reply by hlj24** [reply]
Thank you so much @gabestein -- I'll see about resubmitting the others as well.
